gpg-interface.c

```diff
@@ -88,6 +88,9 @@ int check_signature(const char *payload, size_t plen, const char *signature,
 
 void print_signature_buffer(const struct signature_check *sigc, unsigned flags)
 {
+	if (flags & GPG_VERIFY_QUIET)
+		return;
```

Comment:
It is questionable if `GPG_VERIFY_QUIET` needs to be checked bitwisely since it overrides the other flags (`GPG_VERIFY_RAW`, `GPG_VERIFY_VERBOSE`) anyway.